systems/FederatedProperties/06-Runtime_View.md

```diff
@@ -0,0 +1,18 @@
+# Runtime View
+
+See [3rd party Editor's use case](../Repository/01-Introduction.md#3rd-party-editor): _Extend a non Wikimedia Wikibase with relevant entities from Wikidata_
+
+## Listing federated and local properties in a local Item's statement edit view
+
+```mermaid
+sequenceDiagram
+    autonumber
+    3rd party Editor->>+Item Page: Navigate to Item page
+    3rd party Editor->>+Item Page: Click add a new statement<br> and start typing
+    Item Page->>+Search Properties: Search properties<br> based on user input
+    Search Properties->>+Local Property Search: Search properties<br> in local repository
+    Local Property Search->>+Item Page: Local properties matching the search input
+    Search Properties->>+Remote Property Search: Search properties in remote repository
+    Remote Property Search->>+Item Page: Remote properties matching the search input
```

Comment:
I would have drawn arrows back from "Local Property Search" and "Remote Property Search" to "Property Search", and then from there have an arrow saying "Combined local and remote search results" back to "Item Page". IMO that would make it clearer that as far as the UI is concerned, there is one search request with one list of search results.

Reply:
makes sense, change it.